api/app/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -74,6 +88,22 @@ export async function sandboxGetDocRefsAndUpsert({
     }
   }
 
+  await updateDocQuery({
+    patient: { id: patient.id, cxId: patient.cxId },
+    downloadProgress: {
+      total: entries.length,
+      status: "completed",
+    },
+    ...(convertibleDocCount > 0
+      ? {
+          convertProgress: {
+            status: "processing",
+            total: convertibleDocCount,
+          },
+        }
+      : undefined),
```

Comment:
When does this moves to `completed`, since we're not getting notified by lambdas?

Reply:
I thought we were since we call the sqs-to-fhir lambda and on success it sends to the api

Reply:
Gotcha. For a minute I thought we were inserting on the FHIR server directly, but no. Cool!